tests/sapt-dft-api/input.dat

```diff
@@ -0,0 +1,41 @@
+#! SAPT(DFT) aug-cc-pVDZ interaction energy between Ne and Ar atoms.
+
+Eref = {"SAPT ELST ENERGY":  -0.10190449, # TEST
+        "SAPT EXCH ENERGY":   0.36545706, # TEST
+        "SAPT IND ENERGY":    0.00349281, # TEST
+        "SAPT DISP ENERGY":  -0.24398704, # TEST
+        "CURRENT ENERGY":     0.02305835} # TEST
```

Comment:
actually has to be `#TEST` for the regex to heed the signal, in case you end up making other changes.

Reply:
Fixed, thanks.